packages/init/test/SliceMachineInitProcess-loginAndFetchUserData.test.ts

```diff
@@ -50,8 +50,7 @@ const loginAndFetchUserDataWithStdin = async (
 	await new Promise((res) => setTimeout(res, 50));
 
 	const port: number =
-		// @ts-expect-error - Mocked instance
-		spiedManager.user.getLoginSessionInfo.results[0][1].port;
+		spiedManager.user.getLoginSessionInfo.mock.results[0].value.port;
 
```

Comment:
Is this one oof the small changes between jest and vitest ? `mock.calls` vs `mock.results`?